modules/wallet/update.go

```diff
@@ -546,6 +527,10 @@ func (w *Wallet) ProcessConsensusChange(cc modules.ConsensusChange) {
 		w.log.Severe("ERROR: failed to update consensus change ID:", err)
 		w.dbRollback = true
 	}
+	if err := dbPutConsensusHeight(w.dbTx, cc.BlockHeight); err != nil {
```

Comment:
do we still need this DB value at all?

Reply:
We could refactor it out and load it from consensus initially, but the wallet still pulls it from the database with `dbGetConsensusHeight` in a few places.